auth0/data_source_auth0_client.go

```diff
@@ -0,0 +1,50 @@
+package auth0
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"gopkg.in/auth0.v5"
+	"gopkg.in/auth0.v5/management"
+)
+
+func newDataClient() *schema.Resource {
+	clientSchema := datasourceSchemaFromResourceSchema(newClient().Schema)
+	delete(clientSchema, "client_secret_rotation_trigger")
```

Comment:
We're currently excluding the `client_secret_rotation_trigger` property when building from the client resource schema. This was proposed in #363 and I'm assuming that the exclusion is still valid but need to verify that no other properties should also be excluded.